### DISCUSSION TITLE
Error creating air file could not generate timestamp: Connection reset

### DISCUSSION THREAD

**Discussion by NeronBurningRome** [airsdk/Adobe-Runtime-Support#3550] [Q&A]
I'm working on Flash Builder 4.7. I used to be able to export air file with certificate. But now it when I try to export release build it shows an error: Error creating air file could not generate timestamp: Connection reset. I know that when certificate expires Flash Builder mentions that explicitly during export. So it's not expired certificate.
What else might the problem here?

**Comment by itlancer** [comment]
@NeronBurningRome
Try with AIR SDK 51.1.1.4 or later.
https://github.com/airsdk/Adobe-Runtime-Support/issues/3419

> Github-3419: Correcting default timestamp URL from symantec to digicert

**Reply by NeronBurningRome** [reply]
Thanks. 
Right now I'm still working on SDK 30 :/ Right now I don't have much time to update the SDK, there will be probably a lot of problems with it and I need to release new version soon.
I've seen that people can set TSA in IneliJ IDE, but I can't find this option in Flash Builder. Is it possible to change somehow temporarily the TSA while exporting project in Flash Builder?

**Reply by ajwfrost** [reply]
You could try editing your hosts file to redirect the timestamp server request to the newer location?
i.e. add:
```
216.168.244.9 sha256timestamp.ws.symantec.com
```
On Windows, this file is at c:\windows\system32\drivers\etc\hosts

thanks

**Reply by NeronBurningRome** [reply]
Great, it worked, thanks.
